### ISSUE TITLE
make:test and shorter queries

### ISSUE THREAD

**Issue by sciasfreelancer** [laravel/framework#10262]
1. http://i.imgur.com/G5pDz1G.png Make:test after upgrading 5.0 to 5.1

Just noticed this after upgrading to 5.1 and yes I DID follow the instructions for upgrade. There are indeed more commands missing.

2. ~~http://i.imgur.com/UdflH0i.png Shorter Query~~

~~Possibly not a bug, but would be a great suggestion for laravel to write easier sql queries with same base but different params when it is required multiple rows.~~

**Comment by lukasgeiter** [comment]
You have to `clone` the query builder instance.

**Comment by sciasfreelancer** [comment]
```
$row = new StdClass;
$row->test1 = "test";
$row->test2 = "test2";

$row1 = clone $row;
$row1->test3 = "test3";

$row2 = clone $row;
$row2->test4 = "test4";

var_dump($row, $row1, $row2);
```
Well that works, but what about the missing commands from artisan ?

**Comment by GrahamCampbell** [comment]
`make:test` isn't available in 5.1 yet. It's coming in the next tag.

**Comment by sciasfreelancer** [comment]
So let me get this straight. You are saying the command isn't available in 5.1, but documentation states it is in 5.1 ?

**Comment by Sladewill** [comment]
It will be available in 5.1 when it is tagged by Taylor.
